scripts/qa_crowbarsetup.sh

```diff
@@ -2450,6 +2450,12 @@ function enable_ssl_generic()
             fi
             $p "$a['api']['protocol']" "'https'"
         ;;
+        manila)
+            if [[ ! $(iscloudver 7plus) ]] ; then
```

Comment:
@Itxaka sometimes PRs get reviewed and merged before I can bitch about them ;-) IMHO we should fix both instances for clarity.